src/applications/disability-benefits/all-claims/config/submitForm.js

```diff
@@ -30,13 +30,13 @@ const submitFormFor = eventName =>
         } else {
           let error;
           if (req.status === 429) {
-            error = new Error(`throttled_error: ${req.statusText}`);
+            error = new Error(`vets_throttled_error: ${req.statusText}`);
```

Comment:
is this update for a listener? It looks like the app name gets added to other variations of this and it's also utilized in the forms library with error logic, too
![Screen Shot 2019-07-03 at 2 59 15 PM](https://user-images.githubusercontent.com/13280995/60621519-7fbf8a00-9da3-11e9-8281-cb7111e5c374.png)

Reply:
yes, in forms-system it's looking for `vets_throttled_error` and we didn't have the prefix so it would never get picked up, e.g. always handled as a client Error

Reply:
nice, that's what I thought. this'll actually fix the discovery thing I was looking at, at least partially, I think